style.css

```diff
@@ -658,105 +715,443 @@ body {
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 20px;
+    margin-top: 20px;
+  }
+
+  .ex4 {
+    display: flex;
+    background: url(images/sada2.png);
+    background-size: cover;
+    background-position: center;
+    padding-top: 62px;
+    padding-left: 20px;
+    padding-right: 30px;
+    padding-bottom: 62px;
+    position: relative;
+    margin-bottom: 2%;
+    width: 327px;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-right: 20px;
+    margin-top: 20px;
+    justify-content: flex-end;
+  }
+
+  #r1 {
+    display: flex;
+  }
+
+  #desk_ver {
+    width: 85%;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 10vh;
+  }
+
+  .about_myself {
+    background-image: url(images/abt.png);
+    background-repeat: no-repeat;
+    background-origin: content-box;
+    background-position: 100%;
+    background-attachment: scroll;
+    background-size: 500px;
+    display: flex;
+    flex-direction: column;
+    background-color: #262626;
+    margin-bottom: 0%;
+    padding-bottom: 0%;
+  }
+
+  #abt_mslf {
+    margin: 0%;
+    margin-bottom: 20px;
+    padding: 0%;
+  }
+
+  #abt {
+    margin: 0%;
+    padding: 0%;
+    width: 70%;
+    margin-bottom: 30px;
+  }
+
+  #btn_2 {
+    margin: 0%;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  .text_2 {
+    width: 50%;
+    margin: 20vh;
+  }
+
+  #main_ul {
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+  }
+
+  .skills {
+    background-color: #262626;
+    width: 100%;
+    padding: 0% 0% 100px 0%;
+    margin: 0% 0% 0% 0%;
+  }
+
+  .sklz {
+    display: flex;
+    flex-direction: column;
+    padding: 0%;
+    margin: 0%;
+  }
+
+  .sklz ul {
     display: flex;
+    width: 100%;
+    padding: 0%;
+    margin: 0%;
     flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .j {
+    padding: 10px 0%;
+    margin: 0% 0%;
+  }
+
+  #text {
+    margin: 0%;
+    padding: 0% 0% 20px 0%;
+  }
+
+  #sm1 {
+    display: flex;
+    flex-direction: row;
     justify-content: space-between;
+    margin-bottom: 0%;
+    width: 200px;
+    height: 40px;
+    margin-top: 0%;
+    padding-top: 0%;
     align-items: center;
+    padding-right: 0%;
+    margin-right: 0%;
   }
 
-  ul {
-    list-style: none;
-    margin: 0;
-    padding: 0;
+  .contact {
+    display: flex;
   }
 
-  img {
-    background-color: #3c3a39;
-    border: 0;
+  .part1 {
+    display: flex;
   }
 
-  .btn {
-    border: none;
-    outline: none;
-    background-color: #1c1a19;
+  .text4 {
+    width: 30%;
+    margin: 10vh 0 0 20vh;
+  }
+
+  #headerrr {
+    text-align: left;
+  }
+
+  #form1 {
+    display: grid;
+    width: 50%;
+    margin: 10vh 0 0 10vh;
+  }
```

Comment:
As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 41 54 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/d8d131c1-04da-4d03-9b78-fc1e56980345">
>
> ## Your Design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 20 49 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/555d905d-f232-4ccf-8439-9a7980e89a9b">